hpi-schul-cloud/nuxt-client
# reset scroll status after changing the page

# Short Description

QA note:
Change tested on env:
https://bc-3926-reset-scroll.brb.dbildungscloud.dev/
No issues found. Page is not scrolled down, user see top of the page. 
More info: https://ticketsystem.dbildungscloud.de/browse/BC-3926





## Links to Ticket and related Pull-Requests
https://ticketsystem.dbildungscloud.de/browse/BC-3926


## Changes



## Data-security



## Deployment



## New Repos, NPM packages or vendor scripts



## Screenshots of UI changes



## Checklist before merging


> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed
